partiql-app/partiql-cli/src/main/kotlin/org/partiql/functions/ReadFile.kt

```diff
@@ -12,7 +12,7 @@
  *  language governing permissions and limitations under the License.
  */
 
-package org.partiql.cli.functions
```

Comment:
Everything in the `partiql-cli` gradle project should now be in the `org.partiql.cli` jvm package